src/main/java/com/demonwav/mcdev/platform/canary/CanaryModuleType.java

```diff
@@ -0,0 +1,78 @@
+package com.demonwav.mcdev.platform.canary;
+
+import com.demonwav.mcdev.platform.AbstractModuleType;
+import com.demonwav.mcdev.platform.PlatformType;
+import com.demonwav.mcdev.platform.canary.util.CanaryConstants;
+import com.demonwav.mcdev.platform.canary.util.CanaryLegacyColors;
+import com.demonwav.mcdev.util.CommonColors;
+import com.google.common.collect.ImmutableList;
+import com.intellij.openapi.module.Module;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+import javax.swing.Icon;
+
+public class CanaryModuleType extends AbstractModuleType<CanaryModule> {
+
+    private static final String ID = "CANARY_MODULE_TYPE";
+    private static final CanaryModuleType instance = new CanaryModuleType();
+
+    private CanaryModuleType() {
+        super("net.canarymod", "CanaryLib");
```

Comment:
You can call `this("net.canarymod", "CanaryLib")` instead, and avoid duplicate code in the constructors.